src/ast/converter.js

```diff
@@ -102,7 +102,13 @@ export default class Converter {
 
     const whereClause = this.whereClause(query, null, searchFilter);
 
-    const limitCount = this.limitCount(MAX_TILE_RECORDS);
+    const maxTileLimit = (maxTileRecords && maxTileRecords > MAX_TILE_RECORDS)
```

Comment:
sorry to bring up another thing on this PR, but i'm not sure we want to have a minimum (if i say i only want 500 records per tile, why should i be forced to get 1000?) I think we should just use the param if it is set, otherwise the MAX_TILE_RECORDS default

Reply:
My initial idea was that nobody intentionally would want to go lower than current limit, unless is a mistake. After aligning with Kyle, we will give the possibility to set any param as long is grater than 0. Is the param is not present, we will set the current limit `MAX_TILE_RECORDS`